db/migrate/20200416092429_add_lewagon_username_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddLewagonUsernameToUsers < ActiveRecord::Migration[5.2]
```

Comment:
why are we calling this lewagon username? is it just to test the seed displays the picture correctly?

Reply:
yeah just to show the get profile pics from lewagon alumni list because i created user seed for lewagon alumni and thought better to have realistic profile pics but if we want we can manually upload profile pics on Cloudinary and bind with it

Reply:
this makes sense for now but eventually we can remove any trace of le wagon i dont need to see any of their faces again hahahha

Reply:
jajaja yeah we can change it later